examples/terra-contract/tools/deploy.js

```diff
@@ -0,0 +1,235 @@
+import { LCDClient, MnemonicKey } from "@terra-money/terra.js";
+import {
+  MsgInstantiateContract,
+  MsgMigrateContract,
+  MsgStoreCode,
+} from "@terra-money/terra.js";
+import { readFileSync } from "fs";
+import { Bech32, toHex } from "@cosmjs/encoding";
+import { zeroPad } from "ethers/lib/utils.js";
+import axios from "axios";
+import yargs from "yargs";
+import {hideBin} from "yargs/helpers";
+import assert from "assert";
+
```

Comment:
is this deployment script copied from the other repo ? 

If so, maybe we should release this as a separate js package that we can pull into various projects (and share with the terra ecosystem more broadly). No urgency to do that though.

Reply:
Yes it is, modified it to fit it. 
There are some ways to do it from Terra stuff which are harder. One is terrad which is a nightmare as it's very complex. One is terrain which is new but like anchor it promotes a certain structure of it's own. I add an issue to move to it for later.